src/editor/BspRenderer.cpp

```diff
@@ -1503,6 +1503,11 @@ bool BspRenderer::pickPoly(vec3 start, vec3 dir, int hullIdx, PickInfo& pickInfo
 
 	start -= mapOffset;
 
+	if (!map || map->ents.size() == 0)
+	{
+		return false;
+	}
+

```

Comment:
I think this would prevent you from being able to pick worldspawn polys? Just the check for `!map` should be good.

Reply:
I got crash here when remove all entities from map )  See line https://github.com/wootguy/bspguy/blob/5c7e36d6bab6bd2a83c6f39e5fbc824c382c850e/src/editor/BspRenderer.cpp#L1515 )))

Reply:
Oh, that'll do it. I just tested and am still select faces when all entities are deleted. So, no issue here.

Reply:
Strange...

Reply:
Right, I meant I tested it with your fix and I was wrong that there would be a problem with selecting faces when there's 0 ents.